Framework/Geometry/src/Instrument/ParComponentFactory.cpp

```diff
@@ -92,6 +93,11 @@ ParComponentFactory::create(boost::shared_ptr<const IComponent> base,
   // Everything gets created on the fly. Note that the order matters here
   // @todo Really could do with a better system than this. Virtual function
   // maybe?
+  const StructuredDetector *sd =
```

Comment:
You should merge in master and update this according the the protocol and @MikeHart85 just implemented namely the use of `cloneParameterized` on `IDetectors` see [here](https://github.com/mantidproject/mantid/pull/15716/files)